components/host-dashboard/HostDashboardExpenses.js

```diff
@@ -279,6 +361,24 @@ const HostDashboardExpenses = ({ hostSlug, isDashboard }) => {
           ) : null
         }
       />
+      {expensePipelineFeatureIsEnabled && (
```

Comment:
It really bothers that the `HostInfoCard` resets to the loading state every time we change tabs, maybe updating that logic to check if data is available could fix it?
```jsx
      <Box mb={4}>
        {!data?.host && loading ? (
          <LoadingPlaceholder height={150} />
        ) : error ? (
          <MessageBoxGraphqlError error={error} />
        ) : (
          <HostInfoCard host={data.host} />
        )}
      </Box>
```

Reply:
Fixed this by breaking out the host query into it's own query (that it shares with the view counts) so that it will use the Apollo cache instead of reloading, since the other query changes variables it won't use the cache for that part.

Also fixed a bug Apollo cache update bug where changing the status of an expense would reduce the totalCount for all expense queries (making the "Scheduled for payment" banner lose a count)